novawallet/Modules/Vote/Governance/Model/ReferendumDecidingFunctionProtocol.swift

```diff
@@ -3,6 +3,80 @@ import Foundation
 protocol ReferendumDecidingFunctionProtocol {
     var curve: Referenda.Curve { get }
     func calculateThreshold(for block: BlockNumber) -> Decimal?
+    func calculateThreshold(for delay: Decimal) -> Decimal?
```

Comment:
```calculateDelay```

Reply:
The logic is the same as `calculateThreshold(for block: BlockNumber)`, but instead of passing a block and determining `x` inside, we pass `x` determined by the `delay(for y:)` function.